src/Endpoints/Search.ts

```diff
@@ -0,0 +1,12 @@
+import axiosInstance from "./AxiosConfig"
+import { SearchResponse } from "../Models/Responses/Shared"
+
+
+const baseURL = "https://api.spotify.com/v1/search?"
+
+export async function GetSearchResults(query: string): Promise<SearchResponse> {
```

Comment:
I personally don't like passing in a string here because it means the caller has to create the string themselves. If we're calling this from a component (which we are), it means that we have to put more business logic in components. It can definitely feel like overkill to have a service _and_ an endpoint file for something this simple but it does give the opportunity to separate business and UI logic.